devops/ci.yml

```diff
@@ -100,19 +76,19 @@ jobs:
     displayName: 'CMake'
     inputs:
       cmakeArgs: |
-        .. -GNinja -DCMAKE_BUILD_TYPE=$(BuildType) -DCMAKE_C_COMPILER=$(CC) -DCMAKE_CXX_COMPILER=$(CXX) -DCMAKE_CXX_FLAGS=$(CXXFLAGS) -DENABLE_ASSERTS=ON -DUSE_ASAN=$(Asan) -DVERONA_CI_BUILD=On -DSNMALLOC_CI_BUILD=On -DRT_TESTS=$(RTTests)
+        .. -GNinja -DCMAKE_BUILD_TYPE=$(BuildType) -DVERONA_EXTRA_CMAKE_FLAGS="-DCMAKE_C_COMPILER=$(CC);-DCMAKE_CXX_COMPILER=$(CXX);-DCMAKE_CXX_FLAGS=$(CXXFLAGS)" -DENABLE_ASSERTS=ON -DUSE_ASAN=$(Asan) -DVERONA_CI_BUILD=On -DRT_TESTS=$(RTTests)
 
   - script: |
       set -eo pipefail
-      ninja install
+      ninja
     workingDirectory: build
     displayName: 'Compile'
 
   - script: |
       set -eo pipefail
       export ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-6.0/bin/llvm-symbolizer
       export ASAN_OPTIONS="alloc_dealloc_mismatch=0 symbolize=1"
-      ctest -j 4 -E "([1-9][0-9]00|[4-9]00)" --timeout 400 --output-on-failure
+      ninja check
```

Comment:
On Linux, just calling `ninja` already runs all the tests. Is this step really necessary?

Reply:
That should not be the case.  Except on MSVC, as I couldn't get `ExternalProject_Add_StepTargets` to work correctly with it.

Reply:
On my build, `ninja check` complained the target didn't exist, so I just ran `ninja` and it ran the tests.

Reply:
Okay.  Which type of build did you do for this.  Download, or build for llvm?  Any options?  Which version of CMake?